README.md

```diff
@@ -10,12 +12,21 @@ As a user, you will need to create a new shopping list or join an existing list.
 
 ## Video demo of the app:
 
-[![](https://cdn.zappy.app/33815167c45d74c3ae5af232de633add.png)](https://www.youtube.com/watch?v=mwj74mE9s64&t)
+![](https://github.com/the-collab-lab/tcl-19-smart-shopping-list/blob/read-me/public/smart-shopping-app.gif)
 
 
 ## Live project
 
-The live project can be viewed [here](https://tcl-19-smart-shopping-list-2.web.app/).
+The live project can be viewed [here](https://smart-shopping-list.netlify.app/).
+
+## Dependencies Used
+
+- This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app)
+- [Google Firebase](https://firebase.google.com/) is used to store the data
+- To make integration with Firebase even more straightforward [react-firebase-hooks](https://firebaseopensource.com/projects/csfrequency/react-firebase-hooks/) were installed
```

Comment:
Looks like `react-firebase-hooks` is singular. I think it should be  "was installed" installed instead of "were installed".